modules/tinymce/CHANGELOG.md

```diff
@@ -14,6 +14,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The color preview box in the color picker dialog was not correctly displaying the saturation and value of the chosen color #TINY-6952
 - The color picker dialog will now show an alert if it is submitted with an invalid hex color code #TINY-2814
 
+## 5.7.2 - TBD
```

Comment:
This shouldn't be here, as this will be part of 5.8.0 if we're merging to `develop`. I also don't believe we have any plans to do a 5.7.2 at this stage (or I haven't heard about it) so we should just remove this for now and include this fix in the 5.8 changelog.

- [ ] Move changes into the 5.8.0/unreleased section of the changelog